### PR TITLE
Add Paginator-based cursor pagination

### PR DESCRIPTION
**Pull request by bernardd**
This replaces #127 (due to me no longer having access to the source repo used on that).

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -50,6 +50,10 @@ defmodule AbsintheRelay.Mixfile do
     [
       {:absinthe, "~> 1.5.0 or ~> 1.6.0"},
       {:ecto, "~> 2.0 or ~> 3.0", optional: true},
+      {:paginator,
+        github: "duffelhq/paginator",
+        branch: "master",
+        optional: true},
```
Comment (dolfinus) [inline mix.exs:56]:
```suggestion
      {:paginator, "~> 1.0", optional: true},
```

**Comment by dolfinus** [comment]
@benwilson512 Could you please take a look?

**Comment by benwilson512** [comment]
I don't think this approach is tractable to include in the library first class. There are a variety of paginator libraries out there, I don't want to just pick one and require its use.

However I'd be happy to accept a PR which updates just some of the functions in Absinthe.Relay to make it compatible with using one of these libraries on the side.

**Comment by bernardd** [comment]
That's a reasonable point. I'm no longer working on the project that would have used this, but anyone who can make use of it is more than welcome to use any part of this that might be of use in a better solution :)

**Pull request closed without merging**